engine/gui/src/test/test_gui_script.cpp

```diff
@@ -841,20 +844,20 @@ TEST_F(dmGuiScriptTest, TestCancelAnimation)
         ASSERT_EQ(dmGui::RESULT_OK, result);
 
         int ticks = 0;
-        Vectormath::Aos::Matrix4 t1;
+        dmVMath::Matrix4 t1;
         while (ticks < 4) {
             dmGui::RenderScene(scene, RenderNodesStoreTransform, &t1);
             dmGui::UpdateScene(scene, 0.125f);
             ++ticks;
         }
-        Vectormath::Aos::Vector3 postScaleDiagonal = Vector3(t1[0][0], t1[1][1], t1[2][2]);
+        dmVMath::Vector3 postScaleDiagonal = Vector3(t1[0][0], t1[1][1], t1[2][2]);
 
-        Vectormath::Aos::Matrix4 t2;
+        dmVMath::Matrix4 t2;
         const float tinyDifference = 10e-10f;
         while (ticks < 8) {
             dmGui::RenderScene(scene, RenderNodesStoreTransform, &t1);
             dmGui::UpdateScene(scene, 0.125f);
-            Vectormath::Aos::Vector3 currentDiagonal = Vector3(t1[0][0], t1[1][1], t1[2][2]);
+            dmVMath::Vector3 currentDiagonal = Vector3(t1[0][0], t1[1][1], t1[2][2]);
             if (tinyDifference < Vectormath::Aos::lengthSqr(currentDiagonal - postScaleDiagonal)) {
```

Comment:
Currently, only the types are added to the `dmVMath` namespace.
Another task will be to add the commonly used functions as well.